src/pkg/packager/creator/utils.go

```diff
@@ -5,15 +5,49 @@
 package creator
 
 import (
+	"context"
+	"fmt"
 	"os"
 	"runtime"
 	"time"
 
 	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/pkg/layout"
 	"github.com/defenseunicorns/zarf/src/pkg/packager/deprecated"
+	"github.com/defenseunicorns/zarf/src/pkg/packager/lint"
 	"github.com/defenseunicorns/zarf/src/types"
 )
 
+func validate(createOpts types.ZarfCreateOptions, pkg types.ZarfPackage) error {
+	if err := pkg.Validate(); err != nil {
+		return fmt.Errorf("package validation failed: %w", err)
+	}
+
+	findings, err := lint.Validate(context.TODO(), createOpts)
+	if err != nil {
+		return fmt.Errorf("unable to lint package: %w", err)
+	}
+
+	lint.PrintFindings(findings, types.SevErr, createOpts.BaseDir, pkg.Metadata.Name)
+	if lint.HasErrors(findings) {
```

Comment:
unsure of this is needed if print findings prints (also, should be severr or sevwarn?)

Reply:
I've kept the print as severr intentionally for now, as many of the warnings from lint can be pretty noisy. In the future, I'd like to change some of the lint results to "notes" then print out the lint warnings. 

The table will be printed out, but I want to return an error still that will cause the program to fail if a validate schema is not used. This is what it will look like when someone has a schema error. 

![image](https://github.com/defenseunicorns/zarf/assets/37223396/a4f432b8-7966-4ef2-87d7-0e4ee24dd410)

Reply:
As I think about "errors in package" is probably better for the create case